CSharp/Library/Microsoft.Bot.Connector.Shared/ActivityEx.cs

```diff
@@ -278,7 +278,7 @@ public static TypeT GetChannelData<TypeT>(this IActivity activity)
         /// Check if the message has content
         /// </summary>
         /// <returns>Returns true if this message has any content to send</returns>
-        public static bool HasContent(this IMessageActivity activity)
+        public static bool HasContentExtension(this IMessageActivity activity)

```

Comment:
You can remove the extensions, since IMessageActivity have HasContent() and GetMentions() and the extension methods are redundant.